src/alfasim_sdk/hook_specs_gui.py

```diff
@@ -164,7 +163,7 @@ def alfasim_get_additional_variables():
 
 @hookspec
 def alfasim_get_status(
-    ctx: Context
+    ctx: Context,
 ) -> Optional[List[Union[WarningMessage, ErrorMessage]]]:
```

Comment:
Not really a problem with this.

But if an empty list and a None return are equivalent I will suggest for force the minimum return to be the empty list and remove the `Optional` from the type hint.
Since the sdk is in the `0` series it should be acceptable to break compatibility.

Reply:
Good call